### PR TITLE
PP-5321: renamed config variable

### PR DESCRIPTION
**Pull request by jankowiakmaria**
Renamed environment variable used in the gauge (updated the way it's been used as well: `Duration` -> `int`)
Updated `README.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ The following variables control the background process:
 | Variable | Default | Purpose |
 |---------|---------|---------|
 | `BACKGROUND_PROCESSING_ENABLED` | `true` | enables registering scheduled processes - at the moment it includes only queue based capture methods |
-| `CAPTURE_PROCESS_RETRY_FAILURES_EVERY` | `60 minutes` | this value is used for calculating the metric gauge of messages awaiting capture (not attempted within this interval) |
```
Comment (heathd) [inline README.md:57]: if we're not already committed to this naming, how about something like:

`CHARGES_CONSIDERED_OVERDUE_FOR_CAPTURE_AFTER`

?

Comment (heathd) [reply README.md:57]: please don't change this if it's a lot of work/has interdependencies elsewhere

File: src/main/resources/config/config.yaml [hunk 2]
```diff
@@ -93,15 +93,17 @@ executorServiceConfig:
 captureProcessConfig:
   backgroundProcessingEnabled: ${BACKGROUND_PROCESSING_ENABLED:-true}
 
+  retryFailuresEvery: ${CAPTURE_PROCESS_RETRY_FAILURES_EVERY:-60 minutes}
+  chargesAwaitingCaptureNotProcessedWithinMinutes: ${CHARGES_CONSIDERED_OVERDUE_FOR_CAPTURE_AFTER:-60}
```
Comment (heathd) [inline src/main/resources/config/config.yaml]: we should prob make this consistent

Comment (jankowiakmaria) [reply src/main/resources/config/config.yaml]: Done

**Review by kbottla — APPROVED** [review]
LGTM!

**Comment by jankowiakmaria** [comment]
Thanks!

**Pull request merged**